### ISSUE TITLE
Error: Assertion failed in file src/mpi/coll/helper_fns.c at line 84: FALSE

### ISSUE THREAD

**Issue by freifrauvonbleifrei** [SGpp/DisCoTec#13]
The build on master currently fails with 
```
integration/test_1 2 4
integration/test_1 integration/test_1 2 4
2 4
integration/test_1 integration/test_1 2 4
2 4
integration/test_1 2 4
integration/test_1 integration/test_1 2 4
2 4
integration/test_1 2 4
new manager rank: 2 
new manager rank: new manager rank: 2 
2 
sending class 
manager received status 
sending class 
manager received status 
Assertion failed in file src/mpi/coll/helper_fns.c at line 84: FALSE
memcpy argument memory ranges overlap, dst_=0x7fff9a595c70 src_=0x7fff9a595c68 len_=16
internal ABORT - process 0
scons: *** [distributedcombigrid/tests/test_distributedcombigrid_boost_run] Error 1
```

But it was still working on 8946f8b. These here are the changes that happened since then:

https://github.com/SGpp/DisCoTec/compare/8946f8b...master

It works fine on my system, though!

**Comment by freifrauvonbleifrei** [comment]
Weirdly, this commit  703a90b has working tests on my system, but fails on travis. Maybe something is wrong in the boundary exchange that depends on the MPI version...

 https://travis-ci.org/github/SGpp/DisCoTec/builds/763553872

**Comment by freifrauvonbleifrei** [comment]
fixed the memcpy error with 3172e96fc and the test error with 12c5e7367c .
